controllers/ProjectController.php

```diff
@@ -13,81 +13,89 @@
 use app\notifier\NewProjectNotification;
 use app\notifier\Notifier;
 use Yii;
-use yii\base\InvalidParamException;
+use yii\data\ActiveDataProvider;
+use yii\filters\AccessControl;
 use yii\helpers\ArrayHelper;
 use yii\helpers\Html;
 use yii\helpers\HtmlPurifier;
 use yii\helpers\Markdown;
 use yii\helpers\Url;
 use yii\web\BadRequestHttpException;
 use yii\web\Controller;
-use yii\data\ActiveDataProvider;
-use yii\filters\AccessControl;
 use yii\web\ForbiddenHttpException;
 use yii\web\NotFoundHttpException;
-use yii\filters\VerbFilter;
 use yii\web\Response;
 use yii\web\ServerErrorHttpException;
 use yii\web\UploadedFile;
 
+/**
+ * Class ProjectController
+ *
+ * @package app\controllers
+ */
 class ProjectController extends Controller
 {
+    /**
+     * @return array
+     */
     public function behaviors()
     {
         return [
             'access' => [
                 'class' => AccessControl::className(),
-                'only' => ['create', 'update', 'delete-image'], //only be applied to
+                'only'  => ['create', 'update', 'delete-image'], //only be applied to
                 'rules' => [
                     [
-                        'allow' => true,
+                        'allow'   => true,
                         'actions' => ['create', 'update', 'delete-image'],
-                        'roles' => ['@'],
+                        'roles'   => ['@'],
                     ],
                 ],
             ],
         ];
     }
 
+    /**
+     * @return string
+     */
     public function actionIndex()
     {
+        $limit = Yii::$app->params['project.pagesize'];
+
+        $project          = new Project();
```

Comment:
It is not correct. Model instance should not manage queries, it represents single a row in the database.